src/bindings/typescript/serializers.ts

```diff
@@ -558,9 +558,9 @@ const struct: Serializer<{ name: string; optional: boolean }> = {
     },
     tsType(v) {
         if (v.optional) {
-            return v.name + " | null";
+            return v.name + "_struct | null";
```

Comment:
why this change?

Reply:
Because I've added `_struct` suffix to struct/contract types in TS, so that they don't overlap with contract names

Reply:
Hmmm, this breaks some existing tests (like the ones in `wallet.spec.ts`). I would add a prefix/suffix to a less frequent use case of the contract "struct" (like in the case of a getter returning a contract) to break as little existing test code as possible.

Reply:
Done!